tests/sparseml/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -77,7 +77,7 @@ def test_sparse_model_reload(compressed, config, dtype, tmp_path):
     )
 
     inferred_global_sparsity = SparsityConfigMetadata.infer_global_sparsity(model)
-    assert math.isclose(inferred_global_sparsity, 19.6562, rel_tol=1e-3)
+    assert math.isclose(inferred_global_sparsity, 12.234, rel_tol=1e-3)
```

Comment:
Should we really be checking global sparsity here? or does it make more sense to check tensor sparsity for one of the targeted layers, and assert that the tensor_sparsity `isclose` to the actual sparsity specified in the recipe? To me this seems like a random number that could change if we change the model or make any updates like your diff does?

Reply:
+1!